site/_data/mydoc/mydoc_topnav.yml

```diff
@@ -60,24 +60,24 @@ javadoc_dropdowns:
       version: all
       output: web
       items:
-        - title: lastest
-          external_url: http://edgent.incubator.apache.org/javadoc/latest/index.html
+        - title: latest
+          url: /..\/javadoc\/latest
```

Comment:
When testing locally, Jekyll wasn't recognizing the path without the backslashes. Does `url: /javadoc/latest` work for anyone else?